gdx/src/com/badlogic/gdx/math/Ellipse.java

```diff
@@ -69,6 +69,10 @@ public Ellipse (Vector2 position, float width, float height) {
 		this.height = height;
 	}
 
+	/** Constructs a new ellipse
+	 * 
+	 * @param position Position vector
+	 * @param size Vektor that determinates the size of the ellipse */
```

Comment:
`Vektor` -> `Vector`

Reply:
I think I changed it how you want it to be